common/types/firestore-types.ts

```diff
@@ -2,7 +2,7 @@ import { firestore } from 'firebase/app';
 import { RepeatingPattern } from './store-types';
 
 export type FirestoreCommon = {
-  readonly owner: readonly string[];
+  readonly owner: string | readonly string[];
```

Comment:
I can accept this for now to unbreak dev. But we should really move `owner` out of `FirestoreCommon`. Otherwise I will be able to put a string array into a `FirestoreTag`.